src/plugins/chart_expressions/expression_metric/public/expression_renderers/metric_vis.scss

```diff
@@ -0,0 +1,11 @@
+// Prefix all styles with "mtr" to avoid conflicts.
+// Examples
+//   mtrChart
+//   mtrChart__legend
+//   mtrChart__legend--small
+//   mtrChart__legend-isLoading
+
+.mtrVis {
```

Comment:
Should we use emotion for this instead of scss?

Reply:
Done.